pkg/runtime/cli.go

```diff
@@ -33,7 +31,7 @@ func FromFlags() (*DaprRuntime, error) {
 	appPort := flag.String("app-port", "", "The port the application is listening on")
 	profilePort := flag.String("profile-port", fmt.Sprintf("%v", DefaultProfilePort), "The port for the profile server")
 	appProtocol := flag.String("protocol", string(HTTPProtocol), "Protocol for the application: grpc or http")
-	componentsPath := flag.String("components-path", getDefaultComponentsPath(), "Path for components directory. Standalone mode only")
+	componentsPath := flag.String("components-path", "", "Path for components directory. Standalone mode only")
```

Comment:
Please add a message that when using daprd directly, this path should be provided else no components will be loaded.

Reply:
Done